### ISSUE TITLE
SELinux blocks sounds for TF2

### ISSUE THREAD

**Issue by acerspyro** [ValveSoftware/steam-for-linux#4185]
A lot of voices are blocked by SELinux in TF2 on Fedora 23.

**Comment by Tele42** [comment]
This is a duplicate of #43, the miles audio middleware uses execheap.

**Comment by gdrewb-valve** [comment]
Closing duplicate.
